lib/tasks/projects.rake

```diff
@@ -28,6 +26,10 @@ namespace :projects do
   task :check_status, %i[max_num_of_projects_to_check batch_size] => :environment do |_task, args|
     exit if ENV["READ_ONLY"].present?
 
+    PLATFORMS_FOR_STATUS_CHECKS = %w[pypi npm rubygems packagist cpan clojars cocoapods hackage cran pub elm dub]
+      .map { |platform| PackageManager::Base.format_name(platform) }
```

Comment:
We needed to add case sensitivity checking here, since that was the issue filtering out all records before.